cloud-arm/src/main/java/com/sequenceiq/cloudbreak/cloud/arm/ArmSetup.java

```diff
@@ -61,20 +121,13 @@ public void execute(AuthenticatedContext authenticatedContext, CloudStack stack)
                 syncPollingScheduler.schedule(task);
             }
             client.createContainerInStorage(storageGroup, osStorageName, IMAGES);
-            if (!storageContainsImage(client, storageGroup, osStorageName, stack.getImage().getImageName())) {
-                client.copyImageBlobInStorageContainer(storageGroup, osStorageName, IMAGES, stack.getImage().getImageName());
-                PollTask<Boolean> task = new ArmImageCopyStatusCheckerTask(authenticatedContext, armClient,
-                        new ImageCheckerContext(new ArmCredentialView(authenticatedContext.getCloudCredential()), storageGroup, osStorageName,
-                                IMAGES, stack.getImage().getImageName()));
-
-                syncPollingScheduler.schedule(task);
-            }
         } catch (HttpResponseException ex) {
+            resourceNotifier.notifyAllocation(cloudResource, authenticatedContext.getCloudContext());
```

Comment:
because if the copy operation there is no persisted resource in the db and the resource group remaining in created state on the provider side.